framework/src/main/java/org/checkerframework/framework/flow/CFAbstractAnalysis.java

```diff
@@ -58,12 +60,30 @@
     /** Initial abstract types for fields. */
     protected final List<Pair<VariableElement, V>> fieldValues;
 
+    /** The associated processing environment. */
+    @SuppressWarnings("HidingField")
```

Comment:
Can you add a TODO to the deprecated fields to remove these suppressions when they are removed?

Reply:
@xingweitian TODOs should not be in the JavaDoc.  They should be in `//` comments.  If they are about a `@SuppressWarnings`, then they should appear after annotation, if there is room.  Also @wmdietl asked that you add the TODO to the deprecated field. So this TODO belongs on Analysis#env.